### DISCUSSION TITLE
mitmproxy-rs usage

### DISCUSSION THREAD

**Discussion by louis030195** [mitmproxy/mitmproxy#7207] [Intercept Everything!]
i've been using the python lib to modify all packets going to internet, trying to figure out how to use the rust lib, would appreciate any example or docs

**Comment by mhils** [comment]
mitmproxy_rs is much more low-level and not comparable to mitmproxy (Python). 

API Docs are available at https://mitmproxy.github.io/mitmproxy_rs/, and there are some tests in the repo you can use as examples. That's about it. :)
